plugins/outputs/kinesis/kinesis.go

```diff
@@ -200,7 +200,7 @@ func (k *KinesisOutput) getPartitionKey(metric telegraf.Metric) string {
 		case "static":
 			return k.Partition.Key
 		case "random":
-			u := uuid.NewV4()
+			u, _ := uuid.NewV4()
```

Comment:
Why are we making changes in other plugins?

Reply:
it was giving error that uuid.NewV4() returns 2 values, so i had to make this change for the code to work.

Reply:
Do you have the version of GO that is used by the mainline? Please check.